common/app/views/support/Commercial.scala

```diff
@@ -26,13 +26,19 @@ object Commercial {
       (!isPaidContent && (content.tags.isAudio || content.tags.isVideo))
   }
 
-  def shouldShowAds(page: Page): Boolean = page match {
-    case c: model.ContentPage if c.item.content.shouldHideAdverts => false
-    case p: model.Page if p.metadata.sectionId == "identity" => false
-    case s: model.SimplePage if s.metadata.contentType.contains(Signup) => false
-    case e: model.ContentPage if e.item.content.seriesName.contains("Newsletter sign-ups") => false
-    case _: model.CommercialExpiryPage => false
-    case _ => true
+  def shouldShowAds(page: Page)(implicit request: RequestHeader): Boolean = {
+    if (request.queryString.get("forceads").isDefined) {
+      true
+    } else {
```

Comment:
```suggestion
    ||
```

Reply:
Yeah. I was being really explicit to document the intent but I think you're right that way is still fine.

Reply:
Actually, the suggestion doesn't compile because of the precedence of `||`. It means you also need to wrap the match in a block (or parens). At that point I'm not sure this is any simpler so I think the explicit `if` wins out. Thanks heaps for the review.